src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -151,6 +210,46 @@ const FilterableWorkflowTable = ({
     }
   };
 
+  const includeKeys = useMemo(() => [], []);
+  const excludeKeys = useMemo(() => ['calls'], []);
```

Comment:
Hm, I don't think this is working - when I look at the repsponse we get to this call, I do still see the `calls` data in there.